### DISCUSSION TITLE
Cannot connect more than 8 devices in Repeater mode

### DISCUSSION THREAD

**Discussion by riomccloud** [Mygod/VPNHotspot#407] [Q&A]
Hi!

I'm using a Moto G XT1033 (running Lineage OS 14.1 - Android 7.1.2) as a repeater. Everything works fine, but I noticed that I can't connect more than 8 devices. Does the app limit the number of simultaneous connections? Is it possible to unlock this limit or is it some hardware limitation? Unfortunately a router/repeater is not an option at the moment, as it is a temporary thing.

Thanks!

**Comment by Mygod** [comment]
It might be a limit of your ROM/hardware.
